app/src/main/java/com/ayush/weatherapp/home/HomePresenterImpl.java

```diff
@@ -168,27 +164,38 @@ private void stopLocationUpdates() {
   }
 
   private void fetchAddress(String latLng) {
+    // TODO refactor after mvp complete
+    Disposable disposable = geocodingRepository.getLocationDetails(latLng)
+        .subscribeOn(Schedulers.io())
+        .observeOn(AndroidSchedulers.mainThread())
+        .subscribeWith(getGeoLocationObserver());
```

Comment:
Don't copy paste code from DT. Creating the function serves no useful purpose here.

Reply:
:ok: